project/src/main/java/com/google/sps/data/OpportunitySignup.java

```diff
@@ -0,0 +1,60 @@
+package com.google.sps.data;
+
+/** Class that represents a signup for an opportunity. */
+public final class OpportunitySignup {
+  private String opportunityId;
+  private String email;
+
+  public static class Builder {
+    // Required parameters
+    private String opportunityId;
+    private String email;
+
+    public Builder(String opportunityId, String email) {
+      this.opportunityId = opportunityId;
```

Comment:
Does opportunityId always = eventId? will this cause clashes if multiple opportunities per eventid?